dd-sdk-android/src/test/kotlin/com/datadog/android/utils/forge/ForgeExt.kt

```diff
@@ -24,6 +24,17 @@ import org.json.JSONArray
 import org.json.JSONObject
 import org.junit.jupiter.api.Assumptions.assumeTrue
 
+/**
+ * Will generate an alphaNumericalString which is not matching any values provided in the set.
+ */
+internal fun Forge.aStringNotMatchingSet(set: Set<String>): String {
+    var aString = anAlphaNumericalString()
+    while (set.contains(aString)) {
+        aString = anAlphabeticalString()
+    }
```

Comment:
why are you not generating the string always the same way? Shouldn't it always be `anAlphabeticalString`